src/adapters/pois_styles.js

```diff
@@ -7,5 +7,21 @@ export const filteredPoisStyle = {
     'icon-ignore-placement': true,
     'icon-size': 0.5,
     'icon-anchor': 'bottom',
+
+    'text-font': [ 'Noto Sans Bold' ],
+    'text-size': 10,
+    'text-field': '{name}',
+    'text-allow-overlap': false,
+    'text-ignore-placement': false,
+    'text-optional': true,
+    'text-variable-anchor': ['top', 'bottom-left', 'bottom-right'],
+    'text-offset': [1.5, 0.5],
+    'text-justify': 'auto',
+  },
+  paint: {
+    'text-color': ['case', ['==', ['feature-state', 'selected'], true], '#900014', '#0c0c0e'],
```

Comment:
We should share colors variables in JS too. It's ok for this PR, we need to move forward.

Reply:
Would it be possible to display markers above labels in all cases to workaround the issue ? Lika an order priority. I don't know if it would be ugly.